### PR TITLE
Add tooltips to the UI

### PR DESCRIPTION
**Pull request by Baschtien**
Added tooltips to the most important / unclear labels of the UI. You are all more than welcome to checkout the branch (add_tooltips) and modify/add/delete tooltips, as the ones I've placed are pretty generic and maybe even incorrect.

Note: It is easier to add tooltips with Qt

Warning : Please do not merge this until #576 is merged both into dev and this branch ; there will most probably be conflicts and modifications to be done. (I'll update this message once it is ok)

### CODE REVIEW

File: Holovibes/mainwindow.ui [hunk 1]
```diff
@@ -567,6 +591,9 @@
          </item>
          <item>
           <widget class="QGroupBox" name="ConvoCheckBox">
+           <property name="toolTip">
+            <string>Add a layer of convolution in the postprocessing</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:595]: Add a convolution step in the post-processing stage

File: Holovibes/mainwindow.ui [hunk 2]
```diff
@@ -1152,6 +1185,9 @@
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="QaccLabel">
+             <property name="toolTip">
+              <string>Interval to include after 'z2' (max #img)</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:1189]: Number of frequencies kept by the  SVD/SSA algorithm : the range of frequencies kept is [z2, z2 + width].

File: Holovibes/mainwindow.ui [hunk 3]
```diff
@@ -1162,6 +1198,9 @@
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="PaccLabel">
+             <property name="toolTip">
+              <string>Interval to include after z (max #img)</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:1202]: Number of frequencies kept by the FFT in the time transformation algorithm : the range of frequencies kept is [z, z + width].

File: Holovibes/mainwindow.ui [hunk 4]
```diff
@@ -1045,6 +1072,9 @@
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="PLabel">
+             <property name="toolTip">
+              <string>The Time Transform filter's start frequency</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:1076]: The Time Transform FFT filter's start frequency : the range of frequencies kept is [z, z + width].

File: Holovibes/mainwindow.ui [hunk 5]
```diff
@@ -1094,6 +1124,9 @@
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="Q_Label">
+             <property name="toolTip">
+              <string>The SVD filter's start frequency</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:1128]: The SVD/SSA filter's start frequency : the range of frequencies kept is [z2, z2 + width].

File: Holovibes/mainwindow.ui [hunk 6]
```diff
@@ -2592,6 +2652,9 @@
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="ImportFpsLabel">
+                <property name="toolTip">
+                 <string>The fps to read the file with</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:2656]: The number of images that will be read per second

File: Holovibes/mainwindow.ui [hunk 7]
```diff
@@ -2867,6 +2936,9 @@
                 <layout class="QHBoxLayout" name="numberOfFramesLayout">
                  <item>
                   <widget class="QCheckBox" name="NumberOfFramesCheckBox">
+                   <property name="toolTip">
+                    <string># of frames to record</string>
```
Comment (Hpn4) [inline Holovibes/mainwindow.ui:2940]: Number of frames to record

**Review by Hpn4 — COMMENTED** [review]
- batch_size: "Number of images processed at once by the spatial transformation algorithm"
- output_image_acc: "Number of images averaged together to generate a single output image. It's a post-processing step to remove noise"

**Review by Hpn4 — APPROVED** [review]

**Review by TitouanGragnic — APPROVED** [review]

**Pull request merged**